desktop-ui/settings/settings.hpp

```diff
@@ -66,6 +66,9 @@ struct Settings : Markup::Node {
     bool runAhead = false;
     bool autoSaveMemory = true;
     bool homebrewMode = false;
+#if defined(PLATFORM_WINDOWS)    	
+	bool sBorderless = false;
```

Comment:
Naming convention here does not match the other variables; consider just `bool borderless`

Reply:
Ok. Is changed.